### PR TITLE
Jhalackova podman compose

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Adding managing multi-container apps.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### Is this (based on) existing content?

* old xml:id...
* old URL...

### CODE REVIEW

**Comment by lvicoun** [comment]
[SLE-Micro-compose_en.pdf](https://github.com/SUSE/doc-modular/files/14523255/SLE-Micro-compose_en.pdf)

File: articles/docker-compose.asm.xml [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+	   see https://confluence.suse.com/x/aQDWNg
+      -->
+      <meta name="architecture" content="x86;power" its:translate="no"/>
+      <meta name="productname" its:translate="no">
+        <!-- enter product name and version --><productname version="X.Y">&productname;</productname>
+      </meta>
+      <meta name="title" its:translate="yes">Docker-compose application stack</meta>
+      <meta name="description" its:translate="yes">Overview of docker-compose and its role in
+      creating container-based application steacks.</meta>
+      <meta name="social-descr" its:translate="yes">Creating a container-based application stack.</meta>
```
Comment (dariavladykina) [inline articles/docker-compose.asm.xml:133]:
```suggestion
      <meta name="description" its:translate="yes">Overview of docker-compose and its role in
      creating container-based application stacks</meta>
      <meta name="social-descr" its:translate="yes">Creating a container-based application stack</meta>
```

**Review by dariavladykina — COMMENTED** [review]
Hi Jana, please see some suggestions from me. Thanks!

**Review by danishprakash — COMMENTED** [review]
Looks good barring a few comments, thanks!

File: references/docker-compose-management.xml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    <!--add author's email address-->
+    <meta name="maintainer" content="jsindelarova@suse.com" its:translate="no"/>
+    <abstract><!-- can be changed via merge in the assembly -->
+      <para>
+       The topic covers exiting application stacks, viewing their status and other admnistrating task.
+      </para>
+    </abstract>
+  </info>
+  <para>
+    After you create your application stack, you can use the <command>docker-compose</command>
```
[the remark this answers is not in the document]
Comment (lvicoun) [reply references/docker-compose-management.xml:31]: It is referring to a command, which is docker-compose.

File: articles/docker-compose.asm.xml [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+              tool simplifies the deployment of such an application stack by using an easy-to-use
+              definition file.
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>WHY?</term>
+          <listitem>
+            <para>
+              The article describes how to create a container-based application stack by using &compose;.
```
[the remark this answers is not in the document]
Comment (lvicoun) [reply articles/docker-compose.asm.xml:162]: Why refers to the reason of reading a document.

File: tasks/docker-compose-deploying-application.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    <!-- add author's e-mail -->
+    <meta name="maintainer" content="jsindelarova@suse.com" its:translate="no"/>
+    <abstract><!-- can be changed via merge in the assembly -->
+      <para>
+       The topic covers the command that deploys the &compose; application.
+      </para>
+    </abstract>
+  </info>
+      <para>
+      After you create the proper directory structure and the <filename>compose.yml</filename>
```
[the remark this answers is not in the document]
Comment (lvicoun) [reply tasks/docker-compose-deploying-application.xml:32]: We do avoid using other tenses than present simple on purpose, even though the present perfect is correct here. :)

File: references/docker-compose-management.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (danishprakash) [reply references/docker-compose-management.xml]: We could also use `podman compose` to maintain consistency, it would work just the same imo.

**Pull request merged**